pkg/vsphere/toolbox/service_test.go

```diff
@@ -288,6 +298,17 @@ func TestServiceRunESX(t *testing.T) {
 		}
 	}
 
+	if *testOn == string(types.VirtualMachinePowerStatePoweredOff) {
+		wg.Add(1)
```

Comment:
I've received the PowerOn RPC in every run of that test so far.  But we only need to wait to handle the case of a quick start+kill, so added the timeout just in case.